Gemfile.lock

```diff
@@ -1,39 +0,0 @@
-GEM
```

Comment:
Why is this deleted?

Reply:
Because the lock file generated on Windows is not compatible with the lock file generated on Linux, and vice versa. If you want to use a fixed set of versions, it is better to fix the versions in Gemfile itself.